Firestore/Protos/nanopb/firestore/local/maybe_document.nanopb.c

```diff
@@ -15,7 +15,7 @@
  */
 
 /* Automatically generated nanopb constant definitions */
-/* Generated by nanopb-0.3.8 at Fri Jun  1 18:36:44 2018. */
+/* Generated by nanopb-0.3.9.1 at Tue Aug 14 12:19:56 2018. */
```

Comment:
Is this going to work with the nanopb [0.3.8 that's available via CocoaPods](https://cocoapods.org/pods/nanopb)?

Reply:
It should. Note the check on line 23. That's what nanopb uses to determine if files need to be rebuilt. This assumes nanopb doesn't introduce incompatible changes without incrementing that number.